sway-ir/src/optimize/inline.rs

```diff
@@ -203,6 +203,7 @@ fn inline_instruction(
     if let ValueContent {
         value: ValueDatum::Instruction(old_ins),
         span_md_idx,
+        state_idx_md_idx: current_state_idx_md_idx,
```

Comment:
This is my first time looking at this code so I have no idea what I am looking at.
These abbreviations may be something completely obvious to someone who is not ignorant in this area however that is not the case for newbies.

I generally dislike abbreviations for this reason and think that alternative names should be selected if they are too long when verbose.